deleting-clusters-and-cleaning-up.md

```diff
@@ -0,0 +1,39 @@
+# Deleting clusters and cleaning up
+
+## Deleting clusters
+Clusters managed by Cluster API can be deleted in the same way as any other Kubernetes object.
+
+To delete an individual cluster by name run:
+
+```bash
+kubectl delete cluster docker-cluster-one
+```
+This delete operation ensures that the cluster and all of the objects that form part of the cluster e.g. Machines, Cluster Infrastructure, are correctly deleted. 
+
+To delete all clusters managed by Cluster API run:
+
+```bash
+kubectl delete clusters --all -A
+```
+
+## Cleaning up resources created by this tutorial
+The management cluster in this tutorial is not managed by Cluster API until [you make it self-hosted](./docker-cluster-self-hosted.md).
```

Comment:
The self hosted doc is linked here which currently doesn't lead anywhere until #19 is merged.